platform/broadcom/sonic-platform-modules-dell/z9332f/scripts/z9332f_platform.sh

```diff
@@ -201,12 +201,17 @@ if [ "$1" == "init" ]; then
     platform_firmware_versions
     get_reboot_cause
     echo 1000 > /sys/module/ipmi_si/parameters/kipmid_max_busy_us
+    # Set the PCA9548 mux behavior
+    echo -2 > /sys/bus/i2c/drivers/cls_pca954x/3-0070/idle_state
```

Comment:
Can you take a look at the changes in #2? Are the change being done there compatible with your changes (i.e. if both sets of changes are present, will it work fine)?